src/wp-includes/formatting.php

```diff
@@ -5119,9 +5119,9 @@ function sanitize_option( $option, $value ) {
 	 * @since 2.3.0
 	 * @since 4.3.0 Added the `$original_value` parameter.
 	 *
-	 * @param string $value          The sanitized option value.
-	 * @param string $option         The option name.
-	 * @param string $original_value The original value passed to the function.
+	 * @param mixed $value          The sanitized option value.
+	 * @param string $option        The option name.
+	 * @param mixed $original_value The original value passed to the function.
```

Comment:
The function `sanitize_option` docblock said that it's string do we needs to update that also? 🤔

Reply:
Great idea; done.